File: isegoria/src/main/java/com/eulersbridge/isegoria/models/UserPersonality.java
package com.eulersbridge.isegoria.models;

@SuppressWarnings({"CanBeFinal", "WeakerAccess"})
public class UserPersonality {

    public float agreeableness;
    public float conscientiousness;
    public float emotionalStability;
    public float extroversion;

    // Openness to experiences
    float openess;

    public UserPersonality(float agreeableness, float conscientiousness, float emotionalStability,
                           float extroversion, float openess) {
        this.agreeableness = agreeableness;
        this.conscientiousness = conscientiousness;
        this.emotionalStability = emotionalStability;
        this.extroversion = extroversion;
        this.openess = openess;
    }

}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/model/UserPersonality.kt
package com.eulersbridge.isegoria.network.api.model

data class UserPersonality(
    val agreeableness: Float,
    val conscientiousness: Float,
    val emotionalStability: Float,
    val extroversion: Float,

    // Mis-spelled, matching API mistake
    val openess: Float
)
